scripts/setup.sh

```diff
@@ -75,7 +75,7 @@ download_config() {
 
 fetch_config(){
     if [[ -w ${CONFIG_DIR} ]]; then
-        downloadConfig
+        download_config
```

Comment:
Is this change necessary in this PR?

Reply:
I believe this is invoking the function which now lives in this file (since we changed the set up config in another PR).
I thought it may cause an error in CI/test builds if not amended. However seems to be working fine locally.

Reply:
Ah, yeah – it may be working locally because there are other changes in your git tree that are untracked, or haven't been committed 👍